crowbar_framework/app/models/node_object.rb

```diff
@@ -1210,8 +1224,11 @@ def set_state(state)
       rescue Timeout::Error
         Rails.logger.warn("chef client seems to be still running after 5 minutes of wait; going on with the reboot")
       end
-
-      ssh_cmd("/sbin/reboot")
+      if @node[:platform] == "windows"
+        bmc_cmd("power cycle")
```

Comment:
Instead of adding windows related functionality to `bmc_cmd` method, would it make sense to make make a similar, windows-specific method, say `net_rpc_cmd` and use it here? Imho it'd be cleaner to separate the per-platform functionality into different methods.